lib/kane.ex

```diff
@@ -14,4 +14,15 @@ defmodule Kane do
       "https://www.googleapis.com/auth/pubsub"
   """
   def oauth_scope, do: "https://www.googleapis.com/auth/pubsub"
+
+  @doc """
+  Retrieves the Google Cloud project name from the configured credentials
+  """
+  @spec project :: String.t()
+  def project(goth_config_mod \\ Goth.Config) do
+    account = Application.get_env(:kane, :account, :default)
```

Comment:
Goth uses `:default` internally (see [here](https://github.com/peburrows/goth/blob/v1.2.0/lib/goth/token.ex#L71) and [here](https://github.com/peburrows/goth/blob/v1.2.0/lib/goth/config.ex#L223)), when there is only one account configured. This is not ideal, since it's not public interface, and could change breaking this with a minor or patch version change.

I couldn't find any other way to support existing setups where a specific key is not provided.